tests/test_deepmd_mixed.py

```
@@ -455,5 +455,53 @@ def tearDown(self):
             shutil.rmtree("tmp.deepmd.mixed.single")
 
 
+class TestMixedSystemWithFparamAparam(unittest.TestCase, CompLabeledSys, IsNoPBC):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)